presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -123,6 +123,11 @@ private void doTestReadFullAcid(boolean isPartitioned, BucketingType bucketingTy
             // test filtering
             assertThat(query("SELECT col, fcol FROM " + tableName + " WHERE fcol = 1 ORDER BY col")).containsOnly(row(21, 1));
 
+            onHive().executeQuery(
+                    "INSERT INTO TABLE " + tableName + hivePartitionString + " VALUES (24, 4)");
+            onHive().executeQuery(
+                    "DELETE FROM " + tableName + " where fcol=4");
+
```

Comment:
add an additional verification after that, and before subsequent insert:
```
assertThat(query("SELECT col, fcol FROM " + tableName + " WHERE fcol = 1 ORDER BY col")).containsOnly(row(21, 1));
```

Reply:
added.